opendbc/car/__init__.py

```diff
@@ -25,6 +36,8 @@ class AngleSteeringLimits:
   STEER_ANGLE_MAX: float
   ANGLE_RATE_LIMIT_UP: tuple[list[float], list[float]]
   ANGLE_RATE_LIMIT_DOWN: tuple[list[float], list[float]]
+  MAX_LATERAL_ACCEL: float = MAX_LATERAL_ACCEL  # Used primarily to lower the limits for more conservative approaches.
+  MAX_LATERAL_JERK: float = MAX_LATERAL_JERK  # Used primarily to lower the limits for more conservative approaches.
```

Comment:
What is the point of this? You don't use it? I'd argue you should though, and move the above constants I just commented on to tesla/values.py

Reply:
How does this not fail? These have no defaults